server/src/test-data.js

```diff
@@ -105,7 +105,16 @@ export const loadTestData = () =>
               name: 'VR Wollongong Escarpment',
               details: 'VR operators required to assist with rescue',
               group,
-            }),
+            }).then(event =>
+              Creators.eventResponse({
+                status: 'On Route',
+                detail: 'Attending',
+                destination: 'Scene',
+                eta: 600,
```

Comment:
imo you should always use a timestamp for this. 600 seconds since what? record creation?